ui/imports/shared/controls/WalletAccountListItem.qml

```diff
@@ -0,0 +1,98 @@
+import QtQuick 2.15
+
+import StatusQ 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Utils 0.1 as StatusQUtils
+
+import AppLayouts.Wallet 1.0
+
+import utils 1.0
+
+StatusListItem {
+    id: root
+
+    property bool clearVisible: false
+
+    required property string name
+    required property string address
+    required property string chainShortNames
+    required property string emoji
+    required property string walletColor
+    required property var currencyBalance
+    required property string walletType
+    required property bool migratedToKeycard
+    required property var accountBalance
```

Comment:
It would be nice to have a dedicated Storybook page for `WalletAccountListItem`. E.g. it's not clear for me what's the expected structure `accountBalance`, how it's rendered and if it's really required and may also appear in account selectors.

Reply:
Good point! I've added a storybook page for it.